api/api/controllers/search_controller.py

```diff
@@ -41,12 +41,8 @@
 NESTING_THRESHOLD = config("POST_PROCESS_NESTING_THRESHOLD", cast=int, default=5)
 SOURCE_CACHE_TIMEOUT = 60 * 60 * 4  # 4 hours
 FILTER_CACHE_TIMEOUT = 30
-FILTERED_PROVIDERS_CACHE_KEY = "filtered_providers"
-FILTERED_PROVIDERS_CACHE_VERSION = 2
-THUMBNAIL = "thumbnail"
-URL = "url"
-PROVIDER = "provider"
-QUERY_SPECIAL_CHARACTER_ERROR = "Unescaped special characters are not allowed."
+ENABLED_SOURCES_CACHE_KEY = "enabled_sources"
+ENABLED_SOURCES_CACHE_VERSION = 2
```

Comment:
Thank you for removing the unused constants from [5 years ago](https://github.com/WordPress/openverse/commit/effee2205d1b57d22c90077f8f935cba5e7d1025) :)

Reply:
I didn't even know they were so old, wow! What is strange is that there should be a pre-commit lint step that fails due to unused variables 🤔  Not sure what happened with that...

Reply:
You can't reliably lint for unused variables declared at the module scope in Python, because all top-level variables are automatically exported. It'd be like trying to lint an unused `export const whatever;` in JavaScript. It can't be done unless you statically analyse all references to the module. I don't know if mypy supports something like that, but certainly ruff never could.